Samples/Monofoxe.Samples/Demos/UtilsDemo.cs

```diff
@@ -118,6 +118,18 @@ public UtilsDemo(Layer layer) : base(layer)
 			_stateMachine.AddState(TestStates.Blue, Blue);
 			_stateMachine.AddState(TestStates.Red, Red);
 			// State machine.
+
```

Comment:
Add three-point bezier demo and add a demo that calls GameMath.BezierCurve directly.